srf_generation/source_parameter_generation/gcmt_to_realisation.py

```diff
@@ -96,6 +96,7 @@ def load_args(primary_logger: Logger):
         )
 
     for i, (param_name, filepath) in enumerate(args.source_parameter):
+        filepath = abspath(filepath)
```

Comment:
https://stackoverflow.com/questions/55257955/getting-full-path-from-the-relative-path-with-argparse/55258475#55258475

this is slightly nicer than modifying argparse. I generally don't expect changes to argparse values after parse args. I'm not quite sure how it works for nargs=2 but probably a tuple coming in.

Reply:
The result returned from load_args is the same